packages/rest/src/events/ConnectionEvents.ts

```diff
@@ -0,0 +1,17 @@
+import type { Agent, ConnectionStateChangedEvent } from '@aries-framework/core'
+
+import { ConnectionEventTypes } from '@aries-framework/core'
+import fetch from 'node-fetch'
+
+export const connectionEvents = async (agent: Agent, webhookUrl: string) => {
+  agent.events.on(ConnectionEventTypes.ConnectionStateChanged, async ({ payload }: ConnectionStateChangedEvent) => {
+    const record = payload.connectionRecord
+    const body = record.toJSON()
+
+    await fetch(webhookUrl + '/connections', {
+      method: 'POST',
+      body: JSON.stringify(body),
+      headers: { 'Content-Type': 'application/json' },
+    })
```

Comment:
I think it would be good to extract the functionality into a separate method so it can be used by all the event handlers, this makes it easier to keep them consistent.